spec/features/user_signs_out_spec.rb

```diff
@@ -14,8 +14,9 @@
     user = FactoryGirl.create(:user)
 
     visit new_user_session_path
+    
 
-    fill_in 'Email', with: user.email
+    fill_in 'Login', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.